src/seedu/addressbook/data/person/Address.java

```diff
@@ -27,6 +27,46 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
         }
         this.value = address;
     }
+    
+    public void addressSplit(String address) {
```

Comment:
This method does not seem to be used anywhere. Even though you have new classes Block, Street, Unit, PostalCode, the address is still stored in `this.value`. How can you use the new classes so that the new code is better?